core/settings.py

```
@@ -166,6 +166,15 @@
 PASSWORD_RESET_IP_WINDOW_SECONDS = 900
 PASSWORD_RESET_IP_MAX_REQUESTS = 3
 
+# Analyze Game Rate Limiting
+# Configures the rate limits for the analyze_game_view endpoint.
+# Window duration for tracking analyze game requests
+ANALYZE_GAME_RATE_WINDOW_SECONDS = int(os.environ.get('ANALYZE_GAME_RATE_WINDOW_SECONDS', 60))
```

Ruff (0.15.18)

[warning] Invalid type for environment variable default; expected `str` or `None`

(PLW1508)